pkg/cmd/mom/apply_configuration_command.go

```diff
@@ -0,0 +1,35 @@
+package mom
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/cluster-authentication-operator/pkg/operator"
+
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryapplyconfiguration"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewApplyConfigurationCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryapplyconfiguration.NewApplyConfigurationCommand(RunApplyConfiguration, streams)
+}
+
+func RunApplyConfiguration(ctx context.Context, input libraryapplyconfiguration.ApplyConfigurationInput) (libraryapplyconfiguration.AllDesiredMutationsGetter, error) {
+	// TODO initialize dynamic clients, informers, operator clients, and kubeclients from the input to demonstrate.
```

Comment:
aren't you creating the clients in the `operator.CreateOperatorInputFromMOM` method ?